app/assets/javascripts/modules/remove-filter.js

```diff
@@ -54,9 +54,9 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
          * Just removing " beta " from the haystack would result in
          * "alphagamma", which is wrong.
          */
-        var haystack = ' ' + currentVal.trim() + ' '
+        var haystack = ' ' + currentVal.replace(/^\s+|\s+$/g, '') + ' '
         var needle = ' ' + decodeEntities(removeFilterValue.toString()) + ' '
-        var newVal = haystack.replace(needle, ' ').replace(/ {2}/g, ' ').trim()
+        var newVal = haystack.replace(needle, ' ').replace(/ {2}/g, ' ').replace(/^\s+|\s+$/g, '')
```

Comment:
We're not removing jQuery methods here, this is is JS String.prototype.trim, so can remain.

Reply:
oh right, okay, thanks :) I've dropped the last commit